### ISSUE TITLE
Any chance i could ask a huge favor?

### ISSUE THREAD

**Issue by invisig0th** [radical-cybertools/radical.synapse#14]
Hi!  I'm the maintainer of a current python project named synapse and was wondering if there's any chance you'd be willing to relinquish the pypi package name "synapse".  I would never even assume to ask ( you got there first fair and square :) ) but your "radical.synapse" package seems to have taken over for the role your synapse package used to play.  If not i totally understand but if so, would you mind adding my pypi user "invisigoth" with privs to the "synapse" package on pypi.python.org?

regards,
-visi

**Comment by andre-merzky** [comment]
Hi Visi, 

its funny you ask -- that is exactly how we got the `synapse` pypi token, too, by friendly handover from 'mvillalba' :)  But your observation is correct: we moved our `synapse` package under the `radical` by now, and I'd thus be happy if the package name would get a new useful purpose!  I added you as `owner` - but I don't see an option to remove myself from that role.  If you do find a way to do so, don't hesitate to clean me out, and the other owners, too.  Otherwise, it's all yours!

Best, Andre.
